lib/common/strings.c

```diff
@@ -32,7 +32,7 @@
  *
  * \return Standard Pacemaker return code (\c pcmk_rc_ok on success,
  *         \c EINVAL on failed string conversion due to invalid input,
- *         or \c EOVERFLOW on arithmetic overflow)
+ *         or \c ERANGE if outside long long range)
```

Comment:
`ERANGE` seems to be more in line with common usage, so I'm not sure why I used `EOVERFLOW` a while back.
* https://stackoverflow.com/questions/34981085/whats-the-semantic-difference-between-erange-and-eoverflow
* https://www.gnu.org/software/libc/manual/html_node/Error-Codes.html

Too negative is still considered overflow though, and too small in absolute value (underflow) is impossible for integers (it's either 0 or +/- 1).

Reply:
Yes but I do think ERANGE makes more sense. I think previously we were trying to preserve existing behavior, but that's not a concern now

Reply:
yeah it does, ERANGE is apparently what's normally used for this type of thing. EOVERFLOW is different despite the name